certbot/plugins/common.py

```diff
@@ -262,15 +263,49 @@ def _setup_challenge_cert(self, achall, cert_key=None):
         return response
 
 
-# test utils used by certbot_apache/certbot_nginx (hence
```

Comment:
Rather than just changing this to document production code also not having test coverage, I think it'd be good to write simple unit tests here like was done for the classes above.